src/examples/Makefile.am:62: `pkglibdir' is not a legitimate directory for `PROGRAMS'
--- ecore-1.1.0/src/examples/Makefile.am.orig	2011-11-06 14:09:53.000000000 +0100
+++ ecore-1.1.0/src/examples/Makefile.am	2012-01-01 20:13:32.520328822 +0100
@@ -1,6 +1,6 @@
 MAINTAINERCLEANFILES = Makefile.in
 
-pkglibdir = $(datadir)/$(PACKAGE)/examples
+pkgprgdir = $(datadir)/$(PACKAGE)/examples
 
 filesdir = $(datadir)/$(PACKAGE)/examples
 files_DATA =
@@ -59,7 +59,7 @@
 EXTRA_DIST = $(SRCS) \
 	$(srcdir)/red.png
 
-pkglib_PROGRAMS =
+pkgprg_PROGRAMS =
 
 if EFL_INSTALL_EXAMPLES
 files_DATA += $(SRCS) \
@@ -67,7 +67,7 @@
 endif
 
 if EFL_BUILD_EXAMPLES
-pkglib_PROGRAMS += \
+pkgprg_PROGRAMS += \
 	ecore_animator_example \
 	ecore_fd_handler_example \
 	ecore_poller_example \
